components/base/FilterTabs.tsx

```diff
@@ -94,61 +38,46 @@ export const TabStyle = styled.div`
   }
 `;
 
-const Drop = styled.div`
-  padding: 10px;
+// Tabs are replaced with Select dropdown on mobile
+const SelectWrapper = styled.div`
+  width: 150px;
   @media (${minWidth(BREAKPOINT_MOBILE)}) {
     display: none;
   }
+`;
```

Comment:
Can we add a small amount of bottom padding so this isn't pushed up against the very bottom of the banner?